scripts/installation/install_solc.py

```diff
@@ -1,69 +1,77 @@
 #!/usr/bin/env python3
 
+"""
+Download supported version of solidity compiler for the host operating system.
 
+This script depends on `nucypher.blockchain.eth.sol` submodule and `py-solc-x` library 
+but otherwise does not require installation of `nucypher` or it's dependencies. 
 """
-This file is part of nucypher.
 
-nucypher is free software: you can redistribute it and/or modify
-it under the terms of the GNU Affero General Public License as published by
-the Free Software Foundation, either version 3 of the License, or
-(at your option) any later version.
 
-nucypher is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU Affero General Public License for more details.
+from os.path import dirname, abspath
 
-You should have received a copy of the GNU Affero General Public License
-along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
-"""
+import os
+import sys
+from pathlib import Path
 
 
-"""
-Download supported version of solidity compiler for Linux based operating system. 
-This script depends on `nucypher.blockchain.eth.sol` submodule and `requests` library 
-but does not require installation of `nucypher`. 
-"""
+def get_solc_config_path() -> Path:
+    nucypher = Path('nucypher')
+    sol_package_path = nucypher / 'blockchain' / 'eth' / 'sol'
+    base_dir = Path(dirname(dirname(dirname(abspath(__file__)))))
```

Comment:
An alternative that only uses pathlib:

```suggestion
    base_dir = Path(__file__).resolve().parent.parent.parent
```

Reply:
:heavy_check_mark: